### PR TITLE
UI layout

### PR DESCRIPTION
**Pull request by saketkum11**
- Minimum UI 
- Bootstrap component lib

### CODE REVIEW

**Review by JowelTisso — APPROVED** [review]
* Clean code 
* Looks Good

**Review by aniketxparihar — APPROVED** [review]
Clean Code
Good CSS naming Convention used
Good Code Structure
Liked that you used barrel exports
Good to merge

File: src/Page/Archieve/Archieve.jsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
+import { Card } from "../index";
 const Archieve = () => {
```
Comment (rahulmistry751) [inline src/Page/Archieve/Archieve.jsx:2]: typo error "Archieve".

**Review by rahulmistry751 — APPROVED** [review]
Good folder structure

File: src/Page/Archieve/Archieve.jsx (hunk 1 above)
Comment (saketkum11) [reply src/Page/Archieve/Archieve.jsx:2]: okk sure

**Pull request merged**